doc/Development.adoc

```diff
@@ -0,0 +1,78 @@
+= Development
+
```

Comment:
Let's add a table of content here as well and try to group the development targets into groups